### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by anhmv-1205**
- Init project
- Link: https://edu-redmine.sun-asterisk.vn/issues/9900

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: remove blank

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: reformat code

**Comment by anhmv-1205** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**